fixtures/fixtures.js

```diff
@@ -43,6 +32,14 @@ var Fixtures = {
             {"code": "SOL00000012", "updated_at": "17-05-2017 14:46.15"},
         ]
     },
+    voucherStatus: {
+        "success":
+            {"success":["RVP12345678"], "fail":[], "invalid":[]},
+        "fail":
+            {"success":[], "fail":["FAL11111111"], "invalid":[]},
+        "invalid": 
+            {"success":[], "fail":[], "invalid":["INV1"]},
```

Comment:
Trailing commas are used for 2 reasons:

1- git changes are more accurate when a new item is added to the end of a list (you only see the new line, not the old last line with a comma and the new one.
2- less likely to forget to add one to the line above and cause compile errors when adding to the bottom of the list.

But you are right that is is not needed and the fixtures file has some with and some without.